src/test/resources/role.json

```diff
@@ -148,5 +148,211 @@
 
       }
```

Comment:
We no longer need this role.json in test-resources. Please delete this file, So it will read original role.json from main-resource. otherwise we have to keep update this test-role.json as well.

Reply:
Deleted and tests all running correctly still.